cvat-ui/src/components/header/settings-modal/settings-modal.tsx

```diff
@@ -22,6 +21,8 @@ import PlayerSettingsContainer from 'containers/header/settings-modal/player-set
 import ShortcutsSettingsContainer from 'containers/header/settings-modal/shortcuts-settings';
 import { CombinedState } from 'reducers';
 import { conflict, conflictDetector } from 'utils/conflict-detector';
+import { ImageFilter, ImageFilterAlias } from 'utils/image-processing';
```

Comment:
Settings modal should not know anything about ImageFilters as they are located in others independend components.

Instead, I believe a function to save settings to local storage should be moved to redux actions.

Reply:
Moved to `settings-actions`